lang_python.adoc

```diff
@@ -114,30 +130,33 @@ https://setuptools.readthedocs.io/en/latest/[Setuptools] is the de-facto
 standard way of building and installing python packages. This library
 allows extension with
 https://setuptools.readthedocs.io/en/latest/setuptools.html#extending-and-reusing-setuptools[plugins].
-https://github.com/KOLANICH have developed
-https://github.com/KOLANICH/kaitaiStructCompile.py[an extension to
+https://gitlab.com/KOLANICH has developed
+https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.setuptools[an extension to
 setuptools], allowing you to automate compilation of `*.ksy` files as the
 part of python module build process. The library provides a mostly
 declarative syntax to facilitate installation: you add a parameter
 `kaitai` to your `setup.py` where you enumerate files that you want
 compiled, output directory, compilation flags, and source code
 post-compilation transformations. Here are some examples:
-https://github.com/KOLANICH/NTMDTRead/blob/master/setup.py[1],
-https://github.com/KOLANICH/SpecprParser.py/blob/master/setup.py[2].
+https://gitlab.com/KOLANICH/NTMDTRead/blob/master/setup.py[1],
+https://gitlab.com/KOLANICH/SpecprParser.py/blob/master/setup.py[2],
+https://gitlab.com/KOLANICH/FrozenTable.py/blob/master/setup.py[3],
+https://gitlab.com/KOLANICH/MFCTool.py/blob/master/setup.py[4],
+https://gitlab.com/KOLANICH/RDataParser.py/blob/master/setup.py[5].
```

Comment:
These numbered links look ugly when are in the same line separated by commas (_1, 2, 3, 4, 5_). Another problem with them is that listing "configuration examples" like this _is like a box of chocolates: you never know what you're going to get_. This is quite silly format for the docs of your project: if one would consider using it, is he supposed to open all these examples and manually compare them so he knows how does one use this library in practice?

It would be a lot better if you'd put them into a normal list, one example by line, in format "link - description" (or better "name - description"). Every example should have a unique short description, you don't have to describe the project that the `setup.py` example is from, but rather what features of the `setup.py` provided by your library it uses.

However, I'd rather move these examples into the actual documentation of your [`kaitaiStructCompile.py` repository](https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.py). This could provide you a lot of options how to "sell" these examples - you can paste an example snippet of a possible `setup.py` configuration into the `README.md` of your project, describe in a few sentences what it does, then paste another `setup.py` from another project, point what's different from the previous one, and so on.

This numerical series IMHO isn't much helpful here, and even if it was a "name - description" list, I'm still not sure if it makes sense to have it here in the KS Python notes.

Reply:
thanks, fixed.